metadata-ingestion/setup.py

```diff
@@ -138,7 +138,7 @@ def get_long_description():
 sqlglot_lib = {
     # Using an Acryl fork of sqlglot.
     # https://github.com/tobymao/sqlglot/compare/main...hsheth2:sqlglot:hsheth?expand=1
-    "acryl-sqlglot==18.0.2.dev15",
+    "acryl-sqlglot==18.5.2.dev45",
```

Comment:
Where is this built and published from?

Reply:
see this notion doc https://www.notion.so/acryldata/Updating-acryl-sqlglot-1caa2af9cc2d4e48a63af697be239211?pvs=4